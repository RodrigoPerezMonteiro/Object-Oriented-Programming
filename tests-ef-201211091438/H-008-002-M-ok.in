2
testH8b.dat
5
1
0
0
