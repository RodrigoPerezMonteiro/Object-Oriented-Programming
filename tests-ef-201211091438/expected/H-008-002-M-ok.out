Gestão de uma Cadeia de Restaurantes
1 - Novo
2 - Abrir
3 - Guardar
4 - Guardar Como...
5 - Gestão de Clientes
6 - Gestão de Alimentos
7 - Gestão de Restaurantes
8 - Consultas
0 - Sair
Escolha uma opção: Ficheiro a abrir: Gestão de uma Cadeia de Restaurantes
1 - Novo
2 - Abrir
3 - Guardar
4 - Guardar Como...
5 - Gestão de Clientes
6 - Gestão de Alimentos
7 - Gestão de Restaurantes
8 - Consultas
0 - Sair
Escolha uma opção: Gestão de Clientes
1 - Visualizar clientes
2 - Registar cliente
3 - Alterar tipo
4 - Inscrever em lista
5 - Remover de lista
6 - Listar mensagens
0 - Sair
Escolha uma opção: VEGETARIAN|Pure Vegetarian|a@a.edu|0|0|0
OMNIVOROUS|James T. Kirk|kirk@starfleet.net|0|0|0
OMNIVOROUS|Darth Vader|vader@imperial.net|1|2|6
OMNIVOROUS|Master Yoda|yoda@jedi.edu|0|0|0
Gestão de Clientes
1 - Visualizar clientes
2 - Registar cliente
3 - Alterar tipo
4 - Inscrever em lista
5 - Remover de lista
6 - Listar mensagens
0 - Sair
Escolha uma opção: Gestão de uma Cadeia de Restaurantes
1 - Novo
2 - Abrir
3 - Guardar
4 - Guardar Como...
5 - Gestão de Clientes
6 - Gestão de Alimentos
7 - Gestão de Restaurantes
8 - Consultas
0 - Sair
Escolha uma opção: 